### DISCUSSION TITLE
Can't Import Spacy even after Installing

### DISCUSSION THREAD

**Discussion by Acejoy** [explosion/spaCy#13144] [Help: Installation] [answered]
Hey everyone, I am using a conda environment and I want to install spacy. I used both methods (conda and pip) to install but was not able to import spacy. After installation, when I want to import spacy using the import command, I do not get any return statement , nor do I get any error statement.

1.  I used import statement:
```python
(ml) username@ServerName:~$ python
Python 3.11.5 (main, Sep 11 2023, 13:54:46) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import spacy
^C^C^C



```
2.  I used the command "python -m spacy validate", the output was as such:

```python
(ml) username@ServerName:~$ python -m spacy validate

```
3.  I used the command "python -m spacy info", the output was as such:

```python
(ml) username@ServerName:~$ python -m spacy info

```

Even after waiting for 30 mins, there is no output from both commands. I used conda list to ascertain whether  spacy was installed or not.
I am installing it over a remote server.

Edit:
I tried to create a new environment and tried installing and importing spacy. It worked without any problem. But most of my packages pytorch, tensroboard are in my previous environment, where the program gets stuck at importing spacy . Any help is much appreciated.

**Comment by svlandeg** [comment]
Hi, sorry to hear you've been having issues with this!

> I tried to create a new environment and tried installing and importing spacy. It worked without any problem. 

That's good to hear - it means that at least everything should work by itself, but there's clearly some sort of interaction/interdependency happening in your existing environment.

When you installed spaCy in the existing environment, did you see any warnings/errors? Can you paste the full output log?

Can you also provide us with the output of `conda list` or `pip list` so we can see what else is installed in your environment?

**Reply by Acejoy** [reply, marked as answer]
Thanks for the reply. I solved it reinstalling conda and restarting the server i was using. For some reason, it worked(dont know why tho)
